websites-css/branching.css

```diff
@@ -1,6 +1,6 @@
 .content-section {
-    background-color: var(--color-searchbar);
-    padding: 40px 10vh;
+    background-color: var(--color-content-section);
+    padding: 30px;
     color: white;
```

Comment:
Use the global variables, i think we made one called --color-bg-text that gives white color 😄